app/src/main/java/org/gdg/frisbee/android/pulse/PulseFragment.java

```diff
@@ -156,13 +152,13 @@ public void onPutIntoCache() {
                 }
 
                 @Override
-                public void failure(RetrofitError error) {
-                    if (isAdded()) {
-                        Snackbar snackbar = Snackbar.make(getView(), R.string.fetch_chapters_failed,
```

Comment:
The error message here was "Fetching chapters failed". It does not makes sense in Pulse page. Generic Server error is used.